modules/nf-core/yahs/meta.yml

```diff
@@ -0,0 +1,84 @@
+name: "yahs"
+description: Performs assembly scaffolding using YaHS
+keywords:
+  - scaffolding
+  - assembly
+tools:
+  - "yahs":
+      description: "YaHS, yet another Hi-C scaffolding tool."
+      homepage: "https://github.com/c-zhou/yahs"
+      documentation: "https://github.com/c-zhou/yahs"
+      tool_dev_url: "https://github.com/c-zhou/yahs"
+      doi: "https://doi.org/10.1101/2022.06.09.495093"
+      licence: "['MIT']"
+
+input:
+  # Only when we have meta
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test' ]
+  #
+  - bed:
+      type: file
+      description: BED file containing coordinates of duplicate alignments
+      pattern: "*.{bed}"
+
+  - fasta:
+      type: file
+      description: FASTA reference file
+      pattern: "*.{fasta,fa}"
+
+  - fai:
+      type: file
+      description: index of the reference file
+      pattern: "*.{fai}"
+
+  - if_break:
```

Comment:
Unless my points on `val` inputs are contradicted by someone else, the related documentation will also need to be removed (we defer to the upstream documentation).